api-gateway/security/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.4.1-bin.zip
```

Comment:
Can we bump Gradle to `6.1`?

Reply:
unfortunately it doesn't work with `6.1` some changes need to be made in `knotx-docker` I assume 
```
Script 'https://raw.githubusercontent.com/Knotx/knotx-starter-kit/master/gradle/docker.gradle.kts' line: 105

* What went wrong:
Could not create task ':buildImage'.
> Could not create task of type 'DockerBuildImage'.
   > Could not find method newOutputFile() for arguments [] on task ':buildImage' of type com.bmuschko.gradle.docker.tasks.image.DockerBuildImage.
```